database/changesets/20200317_ExternalWholeGenomeReprocessingTable.xml

```diff
@@ -12,24 +12,19 @@
                id="20200317"
                logicalFilePath="20200317_ExternalWholeGenomeReprocessingTable.xml">
         <comment>
-            Define the ExternalWholeGenomeReprocessing table schema.
-            The name of an instance of this type of table
-            appears in the 'pipeline.table' column.
-            There is a row in this table for every workflow
-            that is part of the workload named by this table.
+            the ExternalWholeGenomeReprocessing table schema
         </comment>
         <sql>
             CREATE TYPE ExternalWholeGenomeReprocessing AS (
-            id                     bigint,      -- unique to this table
-            base_file_name         text,        -- input to the workflow
-            cromwell               text,        -- the Cromwell metadata URL
-            destination_cloud_path text,        -- input to the workflow
```

Comment:
So we don't include final path as an input param anymore?

Reply:
We do, but isn't it shared across all the workflows?